testing/sensortest/sensortest.c

```diff
@@ -37,7 +37,7 @@
  ****************************************************************************/
 
 #define ARRAYSIZE(a)       (sizeof(a) / sizeof(a)[0])
-#define DEVNAME_FMT        "/dev/sensor/%s"
+#define DEVNAME_FMT        "/dev/sensor/sensor_%s"
```

Comment:
revert. let's add prefix "sensor_" for g_sensor_info.name. And using sensor_baro0 to test sensor device.